nucliadb_models/nucliadb_models/search.py

```diff
@@ -728,6 +728,47 @@ class AskDocumentModel(BaseModel):
     user_id: str = Field(description="The id of the user associated to the request")
 
 
+class ContextStrategy(str, Enum):
+    FULL_RESOURCE = "full_resource"
+    EXTEND_WITH_FIELDS = "extend_with_fields"
```

Comment:
Should the conversation auto expansion that we currently do also be considered a context strategy with configuration on how many messages we inject?

Maybe not in this PR I guess though

Reply:
I like the idea, we can do in a separate PR